app/models/likeable.rb

```diff
@@ -0,0 +1,9 @@
+module Likeable
+  extend ActiveSupport::Concern
```

Comment:
Love that we're doing this with concerns. I'd love to make our photos code be a concern too, in future.